0001-customer-onboarding.md

```diff
@@ -0,0 +1,122 @@
+- Feature Name: `scipian_customer_onboarding`
```

Comment:
This first RFC is meant to stay relatively generic, but focus on some internal needs for customers currently using `scipctl` and `stack` who will need to onboard to Scipian. This will include importing existing Terraform state, however this is not a requirement for Scipian, just a scenario we need to support in this first MVP.